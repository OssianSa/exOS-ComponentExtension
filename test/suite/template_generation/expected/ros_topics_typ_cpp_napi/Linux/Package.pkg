<?xml version="1.0" encoding="utf-8"?>
<?AutomationStudio FileVersion="4.10"?>
<Package SubType="exosLinuxPackage" PackageType="exosLinuxPackage" xmlns="http://br-automation.co.at/AS/Package">
  <Objects>
    <Object Type="File" Description="Generated datamodel header for ros_topics_typ">exos_ros_topics_typ.h</Object>
    <Object Type="File" Description="Generated datamodel source for ros_topics_typ">exos_ros_topics_typ.c</Object>
    <Object Type="File" Description="CMake build file script">CMakeLists.txt</Object>
    <Object Type="File" Description="build file shell script">build.sh</Object>
    <Object Type="File" Description="ros_topics_typ N-API wrapper">libros_topics_typ.c</Object>
    <Object Type="File" Description="ros_topics_typ build file">binding.gyp</Object>
    <Object Type="File" Description="ros_topics_typ package information">package.json</Object>
    <Object Type="File" Description="ros_topics_typ package dependency tree">package-lock.json</Object>
    <Object Type="File" Description="ros_topics_typ node module">l_ros_topics_typ.node</Object>
    <Object Type="File" Description="ros_topics_typ debian package">exos-comp-ros_topics_typ_1.0.0_amd64.deb</Object>
    <Object Type="File" Description="ros_topics_typ main javascript application">ros_topics_typ.js</Object>
  </Objects>
</Package>
