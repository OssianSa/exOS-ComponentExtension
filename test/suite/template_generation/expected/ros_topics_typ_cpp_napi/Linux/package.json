{
  "name": "ros_topics_typ_datamodel",
  "version": "1.0.0",
  "description": "implementation of exOS data exchange defined by datatype ros_topics_typ",
  "main": "ros_topics_typ.js",
  "scripts": {
    "start": "node ros_topics_typ.js"
  },
  "author": "your name",
  "license": "MIT"
}
